Python/cp/Cyclically_rotate.py

```diff
@@ -0,0 +1,105 @@
+'''
+To cyclically (clockwise) rotate a list of integers by N
```

Comment:
Write a clearer description of the problem.

Reply:
Alright